meta-service.js

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+/* global $ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.service('brMetaService', factory);
+}
+
+/* @ngInject */
+function factory($compile, $location, $rootScope) {
+  var service = {};
+  var initialized = false;
+
+  service.meta = {};
+
+  service.set = function(meta, options) {
+    options = options || {};
+    var url = options.url || $location.url();
+    service.meta[url] = meta;
+    $rootScope.$emit('brMeta');
+    if(!initialized) {
+      _initBrMeta();
+      initialized = true;
+    }
+  };
+
+  service.update = function(url, property, content) {
+    if(typeof content === 'undefined') {
+      property = url;
+      content = property;
+      url = $location.url();
+    }
+    var meta = service.meta[url] || {};
+    meta[property] = content;
+    service.set(meta, {url: url});
+  };
+
+  function _initBrMeta() {
+    $('head').append($compile('<br-meta></br-meta>')($rootScope));
```

Comment:
Use `angular.element` instead of `$`. This also means including `angular` in the `define` at the top as a dependency.